zaas-service/src/main/java/org/zowe/apiml/zaas/security/config/CompoundAuthProvider.java

```diff
@@ -58,7 +58,10 @@ private void warnForDummyProvider(String defaultProviderName) {
     }
 
     private AuthenticationProvider getConfiguredLoginAuthProvider() {
-        String providerBeanName = loginProvider.getAuthProviderBeanName();
+        String providerBeanName;
+        synchronized (this) {
+            providerBeanName = loginProvider.getAuthProviderBeanName();
+        }
```

Comment:
why is it synchronizing on the instance to update a method local variable?
What was the coverity report for this line?

Reply:
https://cov-mfd-01.devops.broadcom.net:8443/start.html#/reports.htm#v23350/p10285/defectInstanceId=3682882&fileInstanceId=85508229&mergedDefectId=194093